src/components/quickStarts.stories.mdx

```diff
@@ -50,48 +50,64 @@ const MyApp = () => {
 
-1. Trigger the device permission prompt using `deviceLabelTrigger`
+```ts
+// Initialize the `MeetingSession` and invoke device permission prompt to get device permission
+join(
+  meetingSessionConfiguration: MeetingSessionConfiguration,
+  options?: MeetingManagerJoinOptions
+): Promise<void> {};
+
+export interface MeetingManagerJoinOptions {
+  deviceLabels?: DeviceLabels | DeviceLabelTrigger;
+  eventController?: EventController;
+  enableWebAudio?: boolean;
+  activeSpeakerPolicy?: ActiveSpeakerPolicy;
+}
+```
+
+After calling `MeetingManager.join()` method, `MeetingManager` will:
+
+1. Invoke device permission prompt to request device permission
+    * The kind of device for which permission is requested is determined by `MeetingManagerJoinOptions.deviceLabels`
 2. Read the device lists of `AudioInput`, `AudioOutput` and `VideoInput` devices
     * In any case, SDK will read the device list of all three device kinds
     * For a device kind without permission, it may get a device list with blank labels. This result depends on the design of device permission of different browsers
-
 3. Select a device for each kind
 
 ### Suppress Device Permission Prompt
 
-`MeetingManager` provides an optional parameter `deviceLabels: DeviceLabels | DeviceLabelTrigger` in its `join()` method to suppress device permission prompt.
-
-You could either use preset options or customize your own device label trigger.
+By default, after calling `MeetingManager.join()`, the React SDK will request permission for both audio device and video device. You could either use preset options or customize your own device label trigger to suppress this behavior.
 
 #### To Use the Preset Options
 
-1. Call `join()` method with a `deviceLabels` parameter of `DeviceLabels` type to select the devices from which the browser requests permission when joining the meeting
-    * The SDK reads the device list of all kinds of devices, but only selects the device for the kind indicated by `DeviceLabels`
+Pass `DeviceLabels` to the `deviceLabels` property of `MeetingManager.join()`'s optional parameter `options: MeetingManagerJoinOptions`.
 
-```js
-enum DeviceLabels {
-  None,
-  Audio,
-  Video,
-  AudioAndVideo,
-};
+* The SDK reads the device list of all kinds of devices, but only selects the device for the kind indicated by `DeviceLabels`
+
+```ts
+export enum DeviceLabels {
+  None = 1, // Do not invoke device permission prompt - suppress device permission for audio and video device
+  Audio, // The constraint of `getUserMedia` is { audio: true } - suppress device permission request for video device
+  Video, // The constraint of `getUserMedia` is { video: true } - suppress device permission request for audio device
+  AudioAndVideo, // Default value. The constraint of `getUserMedia` is { audio:true, video: true } - no suppression
+}
 ```
 
 For example, a simple App to join a meeting that only requires camera using preset option.
 
-```jsx
+```tsx
```

Comment:
`jsx` is fine as no type usage is shown right?

Reply:
`enum` is from typescript.

```ts
const options = {
  deviceLabels: DeviceLabels.Video,
};
```